scripts/linters/js_ts_linter.py

```diff
@@ -55,6 +55,10 @@
     'CanAccessSplashPageGuard',
 ]
 
+FILES_CONTAINING_UNKNOWN_TYPE: List[str] = [
+    # Do this file after removal Jquery from this file.
+    'extensions/objects/templates/svg-editor.component.ts',
+]
```

Comment:
I think there's supposed to be another blank line after this line. This issue was flagged by the linter (see https://github.com/oppia/oppia/actions/runs/3556542369/jobs/5973984115), but it should have been caught by your pre-push hooks. Did the pre-push hooks run when you pushed commits to this PR?